themes/mlibrary_new/custom.php

```diff
@@ -1,27 +1,102 @@
 ﻿<?php
 require_once dirname(__FILE__) . '/functions.php';
 
- /**
-  * Copyright (c) 2016, Regents of the University of Michigan.
-  * All rights reserved. See LICENSE.txt for details.
-  */
+/**
+* Copyright (c) 2016, Regents of the University of Michigan.
+* All rights reserved. See LICENSE.txt for details.
+*/
 
 // Use this file to define customized helper functions, filters or 'hacks' defined
 // specifically for use in your Omeka theme. Note that helper functions that are
 // designed for portability across themes should be grouped into a plugin whenever
 // possible.
 
 /**
- * Extends featured exhibit function to include snippet from description for exhibits and
- * read more link
- **/
+* Extends featured exhibit function to include snippet from description for exhibits and
+* read more link
+**/
 
 /**
 * Create the information in cards in introduction page, image with title and text
 *
 **/
 
-function get_most_used_tags_in_exhibits()
+function mlibrary_new_exhibit_builder_previous_link_to_exhibit($exhibit, $exhibit_page)
+{
+   $previoustargetPage = $exhibit_page->previous();
+   $navigate_previous_exhibit = ""; 
+   if ($previoustargetPage) {
+      if (!isset($props['class'])) {
+            $props['class'] = 'previous-page';
+      }
+   
+   $link =  exhibit_builder_link_to_exhibit($exhibit, 'Previous Section', $props, $previoustargetPage);
+   $page_card_info = mlibrary_new_display_exhibit_card_info($previoustargetPage);
+   $caption = snippet_by_word_count($page_card_info['description'],10,'..');
+   $navigate_previous_exhibit = array(
+                                 'title'=>$page_card_info['title'],
+                                 'url-link'=>$link,
+                                 'caption'=>$caption,
+                                 'image'=>$page_card_info['image']);
+
+   }
+
+   return $navigate_previous_exhibit;
+}
+
+
+function mlibrary_new_exhibit_builder_next_link_to_exhibit($exhibit, $exhibit_page)
+{
+   $nexttargetPage = $exhibit_page->next();
+   $navigate_next_exhibit = "";
+
+   if ($nexttargetPage) {
+      if (!isset($props['class'])) {
+           $props['class'] = 'next-page';
+      }
+
+   $link = exhibit_builder_link_to_exhibit($exhibit, 'Next Section', $props, $nexttargetPage);
+   $page_card_info = mlibrary_new_display_exhibit_card_info($nexttargetPage);
+   $caption = snippet_by_word_count($page_card_info['description'],10,'..');   
+   $navigate_next_exhibit = array(
+                                 'title'=>$page_card_info['title'], 
+                                 'url-link'=>$link, 
+                                 'caption'=>$caption,
+                                 'image'=>$page_card_info['image']);
+
+   }   
+   
+   return $navigate_next_exhibit;
+}
+
+
+function mlibrary_exhibit_builder_render_exhibit_page($exhibitPage = null)
```

Comment:
Should this have been `mlibrary_new_exhibit_builder_render_exhibit_page()` ?

Reply:
Actually, it is not used yet, I will remove it. I will need it later when I build the new plugin.